### ISSUE TITLE
New app: yarpmotorgui on ROS2

### ISSUE THREAD

**Issue by PeterBowman** [roboticslab-uc3m/ros2-teo-ws#1]
@TheArmega is developing a Python/Qt application with the goal of replicating the main functionalities of yarpmotorgui, using ROS2 as a data carrier instead of YARP. I think this repository might be a good fit, otherwise we could arrange a new one specifically for hosting said app.

**Comment by jgvictores** [comment]
Looking at https://robots.uc3m.es/developer-manual/appendix/repository-index.html#general I think this is okay and a good idea for now. If we develop more in this direction (especially if it involves a ros2-specific folder structure, usually more required in C++ projects), maybe they could spawn off into a new `ros2-tools` repository in the future.

**Comment by TheArmega** [comment]
ASW @PeterBowman we have been discussing about creating a new repository with just the ROS2 MotorGui package. 
If in the future we implemented more modules like this one, we can rename the repository and add them.

cc @jgvictores

**Comment by TheArmega** [comment]
ROS2 Motor GUI finally created!!!

I have upload the ros2 workspace with all the commented software, I have also added an explanation of how to install it, compile it and get it up and running.

Feel free to check out the GUI, test it, and provide feedback. Your input is highly valuable as we continue to improve and refine this component.

cc @jgvictores @PeterBowman

**Comment by PeterBowman** [comment]
Great job! I've tested this and it looks great (together with https://github.com/roboticslab-uc3m/teo-gazebo-models/commit/5d95b9405572e058d18edfd98e1ec2a909930d24 to enable ROS2 NWS devices). A tiny nitpick: I realized the robot port prefix was missing and just hardcoded it at https://github.com/roboticslab-uc3m/ros2-motor-gui/commit/b7b231cc0278c49bd533a522dfe4c5f6f5865535. It would be nice to have a CLI parameter for switching between real and simulated robots (/teo vs /teoSim, but not limited to, i.e. I'd rather pass the prefix instead of a Boolean switch).
